src/client/services/apiCaller/index.js

```diff
@@ -37,6 +37,25 @@ const apiCaller = {
     }).then(res => res.json());
   },
 
+  getLayouts(uri, version) {
+    return fetch(`/api/get-layout-history?${qs.stringify({uri, version})}`, {
+      method: 'GET', headers: {
+        'Content-type': 'application/json',
+        'Accept': 'application/json'
+      }
+    }).then(res =>  res.json());
+  },
+  renderImages(cyJson) {
```

Comment:
Should take in a parameter num > 0 specifying the number of images to get.  E.g get the 10 latest layouts.

The function is also more specific than what it is called.  It should be renderLatestLayoutImages, or something more appropriate.

Reply:
`renderImage()` actually is just a generic renderImages function. All the comments say "recent layouts", but you can pass any set of cytoscape pathways into `renderImages()` to get a rendered image from the server.

The idea is that in the future it could be used for rendering quick previews of graphs to display on the search page.